ProjectCoimbra.UWP/Project.Coimbra/Strings/de-DE/Resources.resw

```diff
@@ -329,24 +329,68 @@
   <data name="LaneSetupControl.KeysSelected" xml:space="preserve">
     <value>Keys {0} selected.</value>
   </data>
+  <data name="OptionsPage.Title.Text" xml:space="preserve">
```

Comment:
The merge seems to have gone a bit strangely here. Ideally, the strings should be sorted alphabetically as I believe VS will automatically do this when it opens the file in the rich string editor. To avoid this, it's better to sort them now and avoid the churn each time the file is hit.